src/Modal/index.jsx

```diff
@@ -10,6 +10,8 @@ import Icon from '../Icon';
 import newId from '../utils/newId';
 import Variant from '../utils/constants';
 
+const isIE11 = !!window.MSInputMethodContext && !!document.documentMode;
```

Comment:
In case anybody from the future wants to know why this works: https://stackoverflow.com/a/22082397/5225575

Reply:
Might not be a bad idea adding the SO link in a comment

Reply:
I'll add a comment here with that link actually.